lib/launchers/ovirt.rb

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env ruby
+require 'ovirtsdk4'
```

Comment:
@santos1709 this doesn't look like a standard ruby gem, correct?  If that's the case, then it needs to be added to the top level Gemfile.